CHANGELOG.md

```diff
@@ -4,6 +4,8 @@
 
 * ![Enhancement][badge-enhancement] Objects that render as equations and whose `text/latex` representations are wrapped in display equation delimiters `\[ ... \]` or `$$ ... $$` are now handled correctly in the HTML output. ([#1278][github-1278], [#1283][github-1283], [#1426][github-1426])
 
+* ![Enhancement][badge-enhancement] Code blocks now support copy to clipboard button to quickly run the code snippet in Julia REPL. ([#1454](github-1454))
```

Comment:
We usually add the original issue reference(s) as well:

```suggestion
* ![Enhancement][badge-enhancement] Code blocks now support copy to clipboard button to quickly run the code snippet in Julia REPL. ([#1055](github-1055), [#1454](github-1454))
```

Will also need a `[github-1055]: https://github.com/JuliaDocs/Documenter.jl/issues/1055` in the link definitions section.